spec/utility/circular_buffer_spec.rb

```diff
@@ -0,0 +1,91 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Bugsnag::Utility::CircularBuffer do
```

Comment:
For new spec files, we should probably use `RSpec.describe` for the top-level one (so we can eventually turn off the global DSL described by https://relishapp.com/rspec/rspec-core/v/3-0/docs/configuration/global-namespace-dsl )

Reply:
Gotcha, I'll change it over.